ui/src/app/shared/service/utils.ts

```diff
@@ -399,6 +399,49 @@ export class Utils {
     }
   }
 
+  public static CONVERT_TO_GRIDBUY_POWER = (value: number | null): string => {

```

Comment:
these constants dont really need to be variables they can be methods which i think is more readable. And the file name utils is not very meaningful shouldnt they be in a file like `Converter` or so?

Reply:
I think that as well. These would include a lot of changes right now, would include that in a future request.